ONSdigital/dp-publishing-dataset-controller
# Add EditMetaData for dataset pages

### What

- Added new endpoint which will get the data needed for the Florence edit metadata screen
- Added functionality to strip requests of credentials which are used to contact the datasetAPI
- New client added for the dataset API
- New mock generated for the dataset API
- New handler created for the edit metadata endpoint
- New mapper created to merge information from the dataset and version endpoint
- New model added that can be converted to JSON
- Status codes on requests now set on requests
- Tests created for the above work

### How to review

- Check that everything runs as expected
- Check that all data needed for the edit metadata screen is available (there is a lot, consult Florence for what it is doing at the moment (this will supersede florences edit meta data code)
- Enusre everything is in the correct format

### Who can review

Anyone except me
